### ISSUE TITLE
SPFX 1.18.2 packages require Node v18

### ISSUE THREAD

**Issue by jimmywim** [SharePoint/sp-dev-docs#9498]
We're seeing lots of BADENGINE warnings when using Node v16 on a project using SPFX 1.18.2, which the page says is supported.

Does the line for 1.18.2 need to say Node v18 only?

```
; node bin location = /opt/hostedtoolcache/node/16.20.2/x64/bin/node
; node version = v16.20.2
; npm local prefix = /home/vsts/work/1/s
; npm version = 8.19.4
; cwd = /home/vsts/work/1/s
; HOME = /home/vsts
; Run `npm config ls -l` to show all defaults.
/opt/hostedtoolcache/node/16.20.2/x64/bin/npm install

added 2790 packages in 1m

242 packages are looking for funding
  run `npm fund` for details
npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: '@azure/core-rest-pipeline@1.13.0',
npm WARN EBADENGINE   required: { node: '>=18.0.0' },
npm WARN EBADENGINE   current: { node: 'v16.20.2', npm: '8.19.4' }
npm WARN EBADENGINE }
npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: '@microsoft/decorators@1.18.2',
npm WARN EBADENGINE   required: { node: '>=18.17.1 <19.0.0' },
npm WARN EBADENGINE   current: { node: 'v16.20.2', npm: '8.19.4' }
npm WARN EBADENGINE }
npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: '@microsoft/eslint-config-spfx@1.18.2',
npm WARN EBADENGINE   required: { node: '>=18.17.1 <19.0.0' },
npm WARN EBADENGINE   current: { node: 'v16.20.2', npm: '8.19.4' }
npm WARN EBADENGINE }
npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: '@microsoft/sp-adaptive-card-extension-base@1.18.2',
npm WARN EBADENGINE   required: { node: '>=18.17.1 <19.0.0' },
npm WARN EBADENGINE   current: { node: 'v16.20.2', npm: '8.19.4' }
npm WARN EBADENGINE }
```

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 25459e4a-2488-302d-9906-cfacfdc4639a
* Version Independent ID: 339fbcec-873c-9c47-fd9c-cf5f505132ac
* Content: [SharePoint Framework development tools and libraries compatibility](https://learn.microsoft.com/en-us/sharepoint/dev/spfx/compatibility)
* Content Source: [docs/spfx/compatibility.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/spfx/compatibility.md)
* Service: **sharepoint**
* Sub-service: **sharepoint-framework**
* GitHub Login: @VesaJuvonen
* Microsoft Alias: **vesaj**

**Comment by gretchunkim** [comment]
No. I'm using  node v17 using SPFx 1.18.1 and also seeing these warning messages.  Hence, these warning messages are not specific to node engine and SPFx. 

Just to remind you, these  are warning messages and compiling solution can continue without issue until these warning messages become error messages.  

Here's another  project example and  that's released  to general SPFx community ( the [masterpiece, PnP Modern Search](https://github.com/microsoft-search/pnp-modern-search/blob/main/search-extensibility/package.json)). The solution works for node >=16.13.0 <17.0.0.   The link should take you to package.json of the PnP Modern Search. 

Based on your snapshot,  you have  azure/core  packages  which is outside of  SPFx dependency, and it's conflict with node 16.   If you really need these outside packages, then  using different version of your package without changing node is the fastest way to troubleshoot.   However, If I were you, I won't resolving every single warning messages due to limited resources. 

Also, you could switch up the node version to troubleshoot. However, you will get separate sets of warning messages.
